src/adc_readout/AdcParse.h

```diff
@@ -184,3 +199,14 @@ TrailerInfo parseTrailer(const InData &Packet, std::uint32_t StartByte);
 /// \return Idle packet timestamp.
 /// \throw ParserException See exception type for possible parsing failures.
 IdleInfo parseIdle(const InData &Packet, std::uint32_t StartByte);
+
+struct ConfigInfo {
+  enum class Version { VER_0, VER_1 } ProtocolVersion;
+  TimeStamp BaseTime;
+};
+
+/// \brief Extract information from the header of a packet for the purpose of
+/// setting up the parsing of subsequent packets. \param[in] Packet Raw data
+/// buffer. \return Packet information, i.e. protocol version and time stamp.
+/// \throw ParserException See exceptions type for possible parsing failures.
```

Comment:
It is perhaps a little more readable if \param always appear at the beginning of the line?

Reply:
I am pretty sure that this is caused by and old version of clang-format as I would never write this myself. I will fix it.